AntiCheat/Lang/en_US.resx

```diff
@@ -238,118 +238,118 @@
     <value>Detected player {player} attempting to call the boss attacking code functions!</value>
   </data>
   <data name="msg_ChatReal" xml:space="preserve">
-    <value>Detected player {player} tried to spoof {player2}'s name!</value>
+    <value>{player} tried to spoof {player2}'s name in chat.</value>
   </data>
   <data name="msg_DespawnItem" xml:space="preserve">
-    <value>Detected player {player} destroying items!</value>
+    <value>{player} was detected destroying items.</value>
   </data>
   <data name="msg_Enemy_ChangeOwnershipOfEnemy" xml:space="preserve">
-    <value>Detected player {player} attempting to change the monster behavior!</value>
+    <value>{player} was detected trying to control a monster.</value>

```

Comment:
its not fully mean the player try to control the monster actually, cheater can make every monster to not targeting any player, that's not Control Company can do, its more than that